app/models/concerns/inquiry/addon/input_setting.rb

```diff
@@ -54,9 +60,16 @@ def validate_select_options
       end
 
       def validate_input_confirm_options
-        if input_type =~ /(select|radio_button|check_box|text_area)/ && input_confirm == 'enabled'
+        if input_type =~ /(select|radio_button|check_box|text_area|upload_file)/ && input_confirm == 'enabled'
           errors.add :input_confirm, :invalid_input_type_for_input_confirm, input_type: label(:input_type)
         end
       end
+=begin
```

Comment:
__[rubocop]__
* Do not use block comments. [:ref](https://github.com/bbatsov/ruby-style-guide#no-block-comments)

<!-- Comment by [SideCI](https://sideci.com) -->